File: code/chapter11/fib.py
import time

def fib(x):
    if x < 3:
        return 1
    return fib(x-1) + fib(x-2)

# start = time.perf_counter()
result = fib(40)
# end = time.perf_counter()

# print(result)
# print(end - start)

File: code/bench/fib.cpp
#include <cstdio>

static long long fib(long long n) {
  if (n < 2) return n;
  return fib(n - 1) + fib(n - 2);
}

int main() {
  long long n = 41;
  long long result = fib(n);
  std::printf("%lld\n", result);
  return 0;
}
